core/src/main/scala/akka/kafka/KafkaConsumerActor.scala

```diff
@@ -127,9 +127,14 @@ private[kafka] class KafkaConsumerActor[K, V](settings: ConsumerSettings[K, V])
       val commitMap = offsets.mapValues(new OffsetAndMetadata(_))
       val reply = sender()
       commitsInProgress += 1
+      val startTime = System.currentTimeMillis()
       consumer.commitAsync(commitMap.asJava, new OffsetCommitCallback {
         override def onComplete(offsets: util.Map[TopicPartition, OffsetAndMetadata], exception: Exception): Unit = {
           // this is invoked on the thread calling consumer.poll which will always be the actor, so it is safe
+          val duration = System.currentTimeMillis() - startTime
```

Comment:
`currentTimeMillis` can't be trusted to be monotonic (system clock adjustments etc.) use `nanoTime` for relative time calculations (although maybe not super important here)

Reply:
nanoTime 👍

Reply:
:+1: will update